### PR TITLE
Make more room for the spectator list

### PR DESCRIPTION
**Pull request by Shosoul**
This change opens up more room for the spectator list to list out names.

### CODE REVIEW

File: mainloop.lua [hunk 1]
```diff
@@ -1690,8 +1690,8 @@ function game_over_transition(next_func, text, winnerSFX, timemax)
 
   while true do
     GAME.match:render()
-    gprint(text, (canvas_width - font:getWidth(text)) / 2, (canvas_height - font:getHeight()) / 2)
-    gprint(button_text, (canvas_width - font:getWidth(button_text)) / 2, ((canvas_height - font:getHeight()) / 2) + 30)
+    gprint(text, (canvas_width - font:getWidth(text)) / 2, (canvas_height - font:getHeight()) / 20)
+    gprint(button_text, (canvas_width - font:getWidth(button_text)) / 2, ((canvas_height - font:getHeight()) / 20) + 30)
```
Comment (JamesVanBoxtel) [inline mainloop.lua:1694]: Why divide by 20? Why not just make this a fixed position?

File: theme.lua [hunk 2]
```diff
@@ -56,15 +56,15 @@ Theme =
...
+    self.win_Pos = {20, 220} -- the position of the win counter
     self.win_Scale = 1 -- the scale size of the win counter
-    self.ratingLabel_Pos = {5, 180} -- the position of the rating label
+    self.ratingLabel_Pos = {5, 140} -- the position of the rating label
     self.ratingLabel_Scale = 2 -- the scale size of the rating label
-    self.rating_Pos = {25, 200} -- the position of the rating value
+    self.rating_Pos = {25, 160} -- the position of the rating value
     self.rating_Scale = 1 -- the scale size of the rating value
-    self.spectators_Pos = {547, 500} -- the position of the spectator list
+    self.spectators_Pos = {547, 460} -- the position of the spectator list
```
Comment (JamesVanBoxtel) [inline theme.lua:67]: Can you provide a screenshot of before and after?

Comment (Shosoul) [reply theme.lua:67]: image
image

Comment (Shosoul) [reply theme.lua:67]: different crop areas oops, but yea

**Review by JamesVanBoxtel — APPROVED** [review]

**Pull request merged**